tensorboard/compat/proto/layout.proto

```diff
@@ -0,0 +1,96 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Can we use the version in the plugin directory?

https://github.com/tensorflow/tensorboard/blob/master/tensorboard/plugins/custom_scalar/layout.proto